maintainers/scripts/gnome.sh

```diff
@@ -170,23 +170,23 @@ if [ "$action" == "update-all" ]; then
       echo "Skipping $project"
     else
       echo "= Updating $project to $majorVersion" >&2
-      update_project $project $majorVersion
+      update_project "$project" "$majorVersion"
       echo >&2
     fi
   done
 else
-  project="$2"
-  majorVersion="$3"
+  project=$2
+  majorVersion=$3
 
   if [ -z "$project" ]; then
     echo "No project specified, exiting" >&2
     usage
   fi
 
-  if [ "$action" == "show" ]; then
-    show_project $project $majorVersion
-  elif [ "$action" == "update" ]; then
-    update_project $project $majorVersion
+  if [ "$action" = show ]; then
+    show_project "$project" "$majorVersion"
+  elif [ "$action" = update ]; then
+    update_project "$project" "$majorVersion"
```

Comment:
…And this admits it can be multiple words. I do not really understand the logic of this cleanup.